articles/azure-monitor/app/asp-net-core.md

```diff
@@ -42,14 +42,12 @@ The [Application Insights SDK for ASP.NET Core](https://nuget.org/packages/Micro
 * **.NET Core version**: All officially [supported .NET Core versions](https://dotnet.microsoft.com/download/dotnet-core) that aren't in preview
 * **IDE**: Visual Studio, Visual Studio Code, or command line
 
-> [!NOTE]
-> ASP.NET Core 6.0 requires [Application Insights 2.19.0](https://www.nuget.org/packages/Microsoft.ApplicationInsights.AspNetCore/2.19.0) or later.
-
 ## Prerequisites
 
 You need:
 
 - A functioning ASP.NET Core application. If you need to create an ASP.NET Core application, follow this [ASP.NET Core tutorial](/aspnet/core/getting-started/).
+- A reference to a recent version of the [Application Insights](https://www.nuget.org/packages/Microsoft.ApplicationInsights.AspNetCore) NuGet package.
```

Comment:
should you mention a minimum version such as 2.21.0 or later?

Reply:
Maybe "support"? All the other versions are marked as deprecated.

```suggestion
- A reference to a supported version of the [Application Insights](https://www.nuget.org/packages/Microsoft.ApplicationInsights.AspNetCore) NuGet package.
```

Reply:
Oh good point. Yeah, supported works.